streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -217,7 +217,7 @@ public StateStore getStateStore(final String name) {
                     forward((ProcessorNode<K, V, ?, ?>) child, key, value);
                 }
             } else {
-                final ProcessorNode<K, V, ?, ?> child = currentNode().getChild(sendTo);
+                final ProcessorNode child = currentNode().getChild(sendTo);
```

Comment:
Rebasing error?

Reply:
This is because I added the typing in line 98 of ProcessorNode below, I will revert it back and update the line below in `forward`.